docs/hub/sample-factory.md

```diff
@@ -0,0 +1,96 @@
+
+All models on the Hub come up with useful features:
+1. An automatically generated model card with a description, a training configuration, and more.
+2. Metadata tags that help for discoverability.
+3. Evaluation results to compare with other models.
+4. A video widget where you can watch your agent performing.
+
+
+## Install the library
+To install the `sample-factory` library, you need to install the package:
+
+`pip install sample-factory`
+
+SF is known to work on Linux and macOS. There is no Windows support at this time.
+
+## Loading models from the Hub
+### Using load_from_hub
+
+To download a model from the HuggingFace Hub to use with Sample-Factory, use the `load_from_hub` script:
+
+```
+python -m sample_factory.huggingface.load_from_hub -r <HuggingFace_repo_id> -d <train_dir_path>
+```
+
+The command line arguments are:
+
+- `-r`: The repo ID for the HF repository to download. The repo ID should be in the format `<username>/<repo_name>`
+- `-d`: An optional argument to specify the directory to save the experiment to. Defaults to `./train_dir` which will save the repo to `./train_dir/<repo_name>`
+
+### Download Model Repository Directly
+
+HuggingFace repositories can be downloaded directly using `git clone`:
+
+```
+git clone <URL of HuggingFace Repo>
+```
+
+## Using Downloaded Models with Sample-Factory
+
+After downloading the model, you can run the models in the repo with the enjoy script corresponding to your environment. For example, if you are downloading a `mujoco-ant` model, it can be run with:
+
+```
+python -m sf_examples.mujoco_examples.enjoy_mujoco --algo=APPO --env=mujoco_ant --experiment=<repo_name> --train_dir=./train_dir
+```
+
+Note, you may have to specify the `--train_dir` if your local train_dir has a different path than the one in the `cfg.json`
+
+## Sharing your models
+### Using push_to_hub
+
+If you want to upload without generating evaluation metrics or a replay video, you can use the `push_to_hub` script:
+
+```
+python -m sample_factory.huggingface.push_to_hub -r <hf_repo_name> -u <hf_username> -d <experiment_dir_path>
+```
+
+The command line arguments are:
+
+- `-r`: The name of the repo to save on HF Hub. This is the same as `hf_repository` in the enjoy script
+- `-u`: Your HuggingFace username
+- `-d`: The full path to your experiment directory to upload
+
+
+### Using enjoy.py
+
+You can upload your models to the Hub using your environment's `enjoy` script with the `--push_to_hub` flag. Uploading using `enjoy` can also generate evaluation metrics and a replay video.
+
+The evaluation metrics are generated by running your model on the specified environment for a number of episodes and reporting the mean and std reward of those runs.
+
+Other relevant command line arguments are:
+
+- `--hf_username`: Your HuggingFace username
+- `--hf_repository`: The repository to push to. The model will be saved to `https://huggingface.co/hf_username/hf_repository`
```

Comment:
This is confusing and inconsistent with the downloading. Shouldn't this be same as download, in which -r was the full id? (username/organization + repository name)

Reply:
What's the repo or PR with this code? I was not able to find anything in the SF repo on the integration

Reply:
@andrewzhang505

Reply:
This has been changed